index.coffee

```diff
@@ -232,3 +223,16 @@ module.exports = class DockerProgress
 
 					onProgress(_.merge(evt, { pushedSize, totalSize, percentage }))
 		)
+
+# Separate string containing registry and image name into its parts.
+# Example: registry.resinstaging.io/resin/rpi
+#          { registry: "registry.resinstaging.io", imageName: "resin/rpi" }
+module.exports.getRegistryAndName = getRegistryAndName = Promise.method (image) ->
```

Comment:
Can you change to:
```
exports.Registry = class Registry
	...
exports.getRegistryAndName = getRegistryAndName = Promise.method (image) ->
	...
exports.Registry = class Registry
	...
exports.DockerProgress = class DockerProgress
	...
```

I know it's a breaking change, but there's existing breaking changes waiting to be published - and this lets us expose all the functionality easily so it can be used (without needed to mess with the `DockerProgress` class)